__init__.py

```diff
@@ -374,8 +379,17 @@ def apply_user_vars(self, user_vars):
             self.define_param("ALT-tcp", ".REMOVE")
             self.define_param("ALT-udp", "")
 
+        # Comment out ZEEK_PLUGIN_PATH in btest.cfg for pure packages.
```

Comment:
This is essentially a hack, lets at least add a comment to motivate it.

```suggestion
        # Comment out ZEEK_PLUGIN_PATH in btest.cfg for pure packages. Since
        # the param is still part of the template we need to emit _something_.
```

Reply:
Yep. I've also looked at the `.REMOVE` approach taken for the `ALT-udp` / `ALT-tcp`, but that felt just as involved.